build-logic/basics/src/main/kotlin/gradlebuild/basics/PublicApi.kt

```diff
@@ -49,4 +49,6 @@ object PublicApi {
     )
 
     val excludes = listOf("**/internal/**")
+
+    val kotlinIncludes = listOf("org/gradle/kotlin/**")
```

Comment:
:x: This is unused now, isn't it?

Reply:
Yes, since you removed its usage. Cleaning it up.